modules/ROOT/partials/configuration/mediaembed_service_url.adoc

```diff
@@ -1,5 +1,19 @@
+[[mediaembed_service_url]]
 == `+mediaembed_service_url+`
 
 This option specifies the URL to the service that will handle your requests and return the embeddable snippets used by the *Media Embed* plugin. Please follow these xref:introduction-to-premium-selfhosted-services.adoc[instructions] to configure the *WAR* file that you will get as a part of your link:{pricingpage}/[{enterpriseversion} subscription]. This option is not required for xref:editor-and-features.adoc[{cloudname}].
 
 Type: `+String+`
+
+Default Value: `+''+`
```

Comment:
```suggestion
```
When it's empty we don't specify a default as it's the same as being nullable in this case.